### DISCUSSION TITLE
HTTPX Showing Error || Error: No such option: --status-code

### DISCUSSION THREAD

**Discussion by shrivastava67** [projectdiscovery/httpx#699] [Q&A] [answered]
Whenever i am trying to run a http response on httpx it shows one error **Error: No such option: --status-code** please help in this.

**Comment by forgedhallpass** [comment, marked as answer]
Please see:

* https://github.com/projectdiscovery/httpx/discussions/775#discussioncomment-3673723
* https://github.com/projectdiscovery/httpx/discussions/774#discussioncomment-3673676
